### ISSUE TITLE
Random Crash  GGSCRP

### ISSUE THREAD

**Issue by yamashi** [imatix/gsl#87]
Hello,

We are getting a random crash on windows only, the crash never occurs in debug builds, only release builds. I have never been able to repro it on my machine, it happens on some of our build machines, randomly, the GSL file causing the crash is also random.
Here is the SMT_DETECT ouput : Abort at GGSCRP:1: (0, 13)

Max

**Comment by evoskuil** [comment]
Not much can be done to help without a repro script.

**Comment by yamashi** [comment]
Yes but the problem is not file related, we found a computer with consistent crash (crashes on all files in release and debug), the output is GGSCRP:Open-The-File: (After-Init, File-Event)

Which seems to mean that it doesn't even get to processing the file..

**Comment by evoskuil** [comment]
Just perusing the source for NDEBUG only issues, I find these:
```c
ASSERT (chdir (workdir));
```
and
```c
ASSERT (dup (file_handle));
```
but the above cases are VMS or UNIX only.

**Comment by evoskuil** [comment]
With a repro scenario you should be able to view the call stack in the debugger.

**Comment by yamashi** [comment]
Yes I will setup a dev environment on the computer and track it down,
Thanks
